### PR TITLE
Development

### PR DESCRIPTION
**Pull request by marcJV**

### CODE REVIEW

File: SKAToolKit/Categories/NSMutableDictionary+SKATMXParserUtils.m [hunk 1]
```diff
@@ -54,151 +54,119 @@
 @implementation NSMutableDictionary (SKATMXParserUtils)
 
 #pragma mark - Public
-- (void)cleanseMap
-{
-    [self cleanseProperties];
+- (void)cleanseMap {
```
Comment (benkane) [inline SKAToolKit/Categories/NSMutableDictionary+SKATMXParserUtils.m]: Our coding standard for this project is to put opening braces on new lines. All of these opening braces should stay how they were.

**Comment by marcJV** [comment]
I'll revert those tomorrow, I took one of the three coding style in there and ran with it

**Comment by benkane** [comment]
Thanks. Good to be consistent. The other ways must have been there by mistake.

**Comment by marcJV** [comment]
We should look into

**Pull request closed without merging**